init.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS
+    Restores NuGet packages.
+#>
+Param(
+)
+
+Push-Location $PSScriptRoot
+try {
+    $HeaderColor = 'Green'
+    $toolsPath = "$PSScriptRoot\tools"
+    $nugetVerbosity = 'quiet'
+    if ($Verbose) { $nugetVerbosity = 'normal' }
+
+    # First restore NuProj packages since the solution restore depends on NuProj evaluation succeeding.
+    gci "$PSScriptRoot\src\project.json" -rec |? { $_.FullName -imatch 'nuget' } |% {
```

Comment:
I am not very good with powershell, but is project.json not located a directory deeper?